core-plugins/src/main/java/co/cask/hydrator/plugin/common/Properties.java

```diff
@@ -156,6 +156,18 @@
   }
 
   /**
+   * Configuration for KinesisSink
+   */
+  public static class KinesisRealtimeSink {
+    public static final String NAME = "name";
+    public static final String BODY_FIELD = "body.field";
```

Comment:
basePath is not external, since TPFS is very much a CDAP dataset too. I agree there should be standardization, but it seems like we use camel case for everything in hydrator.